src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -33,7 +33,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long id = entity.getKey().getId();
       String message = (String) entity.getProperty("message");
       Date timestamp = (Date) entity.getProperty("date");
-      reviews.add(message + " - " + timestamp.toString());
+      int total = (int) entity.getProperty("total");
+      int positive = (int) entity.getProperty("positive");
+      int negative = (int) entity.getProperty("negative");
```

Comment:
you don't use positive or negative inside of the loop

Reply:
Good catch - fixed